Backend/Models/PharmacyEntity/DTO/GetPharmaciesDTO.cs

```diff
@@ -1,7 +1,8 @@
-﻿using Newtonsoft.Json;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
+using Backend.Models.Common;
+using Newtonsoft.Json;
 
```

Comment:
Shouldn't this file be removed? This is not used anywhere and was changed by GetListDTO\<T>

Reply:
You are correct. I have addressed this issue in another branch and it will be fixed with the next MR.